code/modules/projectiles/guns/flamer.dm

```diff
@@ -308,17 +308,20 @@
 	starting_attachment_types = list(/obj/item/attachable/flamer_nozzle/wide, /obj/item/attachable/magnetic_harness)
 
 //dedicated engineer pyro kit flamer
-/obj/item/weapon/gun/flamer/big_flamer/engineer
-	name = "\improper FL-242 incinerator unit"
-	desc = "The FL-242 is a more light weight incinerator unit designed specifically to fit into its accompanying engineers bag. Can only be used with back mounted fuel tanks however."
+/obj/item/weapon/gun/flamer/big_flamer/marinestandard/engineer
```

Comment:
it wasn't supposed to have an extinguisher, otherwise its just better than a normal flamer in basically everyway (it should have nozzles though).